source/2020-01-10-the-ember-times-issue-130.md

```diff
@@ -36,13 +36,30 @@ How to migrate to Ember Octane 🐹🏗️,
 
 ---
 
-## [Section Title in Title Case 🐹](#section-url)
+## [Community Feedback 👏](https://www.outdoorsy.com)
```

Comment:
I liked that we'll dedicate several issues, and look forward to learn how different teams are using Octane! What do you think about adding the team name to the title, to highlight that we'll have multiple coverages?
```suggestion
## [Community Feedback: Outdoorsy 👏](https://www.outdoorsy.com)
```